components/IscnUploadForm.vue

```diff
@@ -334,5 +516,171 @@ export default class UploadForm extends Vue {
     this.displayImageSrc = this.fileRecords[index].fileData
     this.displayExifInfo = this.fileRecords[index].exifInfo
   }
+
+  onOpenKeplr() {
+    logTrackerEvent(this, 'ISCNCreate', 'OpenKeplr', '', 1);
+    this.isOpenKeplr = true
+    setTimeout(() => {
+      this.isOpenKeplr = false
+    }, 5000)
```

Comment:
Seems this is old code, but what does this magic 5s do?

Reply:
sometimes encountering no response from Keplr when calling `isOpenKeplr = true,` so we added a setTimeout to restart it.

Reply:
But I don't know why it works.🌚

Reply:
Maybe add a comment here to explain